src/api/firebase.js

```diff
@@ -30,6 +33,14 @@ const db = getFirestore(app);
  */
 export function streamListItems(listId, handleSuccess) {
 	const listCollectionRef = collection(db, listId);
+	// see https://firebase.google.com/docs/firestore/query-data/get-data
+
+	const querySnapshot = getDocs(listCollectionRef);
+	// but this just gets docs assuming listCollection exists. falsy default if empty.
+
+	// const docRef = doc(db, "tokens", listId)
+	// const docSnap = getDoc(docRef)
+	// console.log(`${docSnap.exists() ? `${listId} was found in collection` : `${listId} not found`}`)
```

Comment:
Wondering what y'all are testing here with this code? Just curious

Reply:
I think probably @h-yung was testing here to be able to check whether collections are empty or not (we found out later in docs that that wasn't possible though). This comment probably needs to be taken out!